tests/Moq.Tests/Matchers/Wildcard.cs

```diff
@@ -63,11 +60,11 @@ int ISomeService.UseInterface(ISomeService a)
 			throw new NotImplementedException();
 		}
 
-		public static implicit operator int(AnyValue _) => It.IsAny<int>();
-		public static implicit operator byte(AnyValue _) => It.IsAny<byte>();
-		public static implicit operator GearId(AnyValue _) => It.IsAny<GearId>();
-		public static implicit operator Animal(AnyValue _) => It.IsAny<Animal>();
-		public static implicit operator Dolphin(AnyValue _) => It.IsAny<Dolphin>();
+		public static implicit operator int(AnyValue _) => default;
+		public static implicit operator byte(AnyValue _) => default;
```

Comment:
@stakx should I switch these back to using `It.IsAny<T>()`? Or should I rely on implicit conversion functions like this never being called? Just thinking about making a source generator for people to use and if possible, I don't want to rely on behavior that may be changed.

Reply:
No, keep these as `default`. You should only have one call to a matcher per parameter. If you have more than that, you may confuse the `SetupSet` / `VerifySet` delegate reconstruction machinery ([`ActionObserver`](https://github.com/moq/moq4/blob/main/src/Moq/ActionObserver.cs)).